### PR TITLE
Concept Coach Teacher Dashboard

### PR DESCRIPTION
**Pull request by jivewise**
screen shot 2015-11-18 at 6 17 55 pm

- [x] Make route and component for initial dashboard display
- [x] Make request to backend for dashboard json
- [x] Show allow changing of periods
- [x] Show tasks
- [x] Show task completion and CC data
- [x] Chapter/section sorting - might need more chapters/sections in data to test
- [x] Link to detailed scores
- [x] Empty period messages
- [x] Linking to correct dashboard when there's only one concept coach course

Still need to do in a separate PR:
- [ ] Spaced practice progress bar (not in api)
- [ ] Update completed progress bar to be like latest in mockup
- [ ] Book pdf/webview links

### CODE REVIEW

**Comment by jason-holmes** [comment]
One change...for the student complete, in progress, not started graph, we talked Kathi into a just doing a % complete.  Here's a link to the visual design for the teacher dashboard.  http://3lwsnp.axshare.com/#p=teacher_dashboard_-_visual_design

File: resources/styles/components/cc-dashboard/index.less [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.dashboard-table-header {
+	font-size: 1.2rem;
```
Comment (philschatz) [inline resources/styles/components/cc-dashboard/index.less]: 8 spaces for indents? Could it be 2 instead?

**Comment by philschatz** [comment]
Otherwise, the code looks really clean :smile:

File: resources/styles/components/cc-dashboard/index.less (hunk 1 above)
Comment (jivewise) [inline resources/styles/components/cc-dashboard/index.less]: Crap, sorry, some vim defaults there. I'll update...

**Pull request merged**